### ISSUE TITLE
Improper `OSC 52` output

### ISSUE THREAD

**Issue by Fan-iX** [netxs-group/vtm#453]
bash command

```
printf '\e]52;c;MTIz\a' # 'MTIz' is the base64 encode for '123'
```

set (Windows) clipboard to:

`123` in vtm v0.9.9h (proper behavior)

`c;123` in vtm v0.9.14 (extra `c;` prefix)

**Comment by o-sdn-o** [comment]
Thanks! I’ll fix it.

**Comment by o-sdn-o** [comment]
I fixed the clipboard synchronization. Could you please test this.

**Comment by o-sdn-o** [comment]
At first glance, everything works fine. Feel free to reopen this issue if something wrong.

**Comment by Fan-iX** [comment]
Well, for the updated version, I simply receive no `OSC 52` output (no clipboard data is set) from the bash command `printf '\e]52;c;MTIz\a'` in Windows terminal 😂

**Comment by o-sdn-o** [comment]
I can't repro.

I run WT with pwsh profile, type bash, paste `printf '\e]52;c;MTIz\a'`, press Enter, press Ctrl+V - got `123`.

**Comment by o-sdn-o** [comment]
An extra semicolon crept in when sending the clipboard to Linux...
```
      443: stdout: 
      443:      \e]52;c;;MTIz\u0007
      443:      \e]2;root@megasus: ~\u0007
```

**Comment by o-sdn-o** [comment]
Fixed in v0.9.16.

**Comment by Fan-iX** [comment]
v0.9.16 works fine
